### DISCUSSION TITLE
sdrpp / gpredict / rigctl / recorder - AOS/LOS

### DISCUSSION THREAD

**Discussion by HemiBob** [AlexandreRouma/SDRPlusPlus#1080] [Q&A]
I'm using gpredict to track NOAA sats and set sdr++ to the correct frequency and to trigger recording of the audio. However, whilst AOS will start a recording, LOS does not stop it so it records for hours until manual intervention. Frequency control works well.

rigctl Server is set up for recording and is connected on port 4532.
gpredict is set to signal AOS and LOS on port 4532.

Is there a simple way I can monitor the traffic on port 4532, maybe pipe to a text file?
ATM I don't know if gpredict is failing to send the signal or sdr++ is not responding to it.

Linux Mint 21.1 Vera
gpredict - ver 2.3-72-gc596101-3
sdr++ ver 1.1.0-1018
RSPdx with the latest sdrplay API

Thanks guys.

![aos_los](https://github.com/AlexandreRouma/SDRPlusPlus/assets/25841598/6487947e-fc7e-4cad-a9d0-7e6ff0cc7dfc)
![rigctl2](https://github.com/AlexandreRouma/SDRPlusPlus/assets/25841598/86cef73d-59d5-4618-a7fd-299e92c059ff)

**Comment by shokifrend007** [comment]
If you still have the problem you can check the rigctl commands that are being sent to sdr++ by running `sdrpp` in a terminal.
In the messages you should see `Rigtctl command: 'AOS'` and `Rigtctl command: 'LOS'` if Gpredict is working properly.
For me your setup is working fine.
